### PR TITLE
Re-adding PHE model

### PR DESCRIPTION
**Pull request by andrewlee94**
## Summary/Motivation:
In the re-organization it seems the plate heat exchanger model fell through a crack and didn't get moved to its new location (and was thus deleted). This PR restores it.

## Changes proposed in this PR:
-
-

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: docs/reference_guides/model_libraries/models_extra/phe.rst [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+The following variables are declared in addition to those variables created by the HeatExchangerNTU class.
+
+=========================== ================== =========== =============================================================================
+Variable                    Symbol             Index Sets  Doc
+=========================== ================== =========== =============================================================================
+plate_length                :math:`L`          None        Length of a heat exchanger plate
+plate_width                 :math:`W`          None        Width of a heat exchanger plate
+plate_thickness             :math:`H`          None        Thickness of a heat exchanger plate
+plate_pact_length           :math:`L_{pact}`   None        Compressed plate pact length
```
Comment (eslickj) [inline docs/reference_guides/model_libraries/models_extra/phe.rst:81]: This is probably just my ignorance, but what's "pact?"

**Review by eslickj — APPROVED** [review]

File: docs/reference_guides/model_libraries/models_extra/phe.rst [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+.. math::
+  \Delta P = \frac{2fN_{passes}v^2\rho_{mass} \times (L_{plate} + d_{port})}{d_{channel}} + 0.7N_{passes}v^2\rho_{mass}g \times (L_{plate} + d_{port})
+
+where :math:`f` is the friction factor for the side and :math:`v` is the velocity of the fluid at the port.
+
+PlateHeatExchanger Class
+------------------------
+
+.. module:: idaes.power_generation.carbon_capture.mea_solvent_system.unit_models.plate_heat_exchanger
```
Comment (eslickj) [inline docs/reference_guides/model_libraries/models_extra/phe.rst:152]: Moved

**Review by anujad95 — DISMISSED** [review]

**Review by aostace01 — DISMISSED** [review]

File: docs/reference_guides/model_libraries/models_extra/phe.rst (hunk 1 above)
Comment (andrewlee94) [reply docs/reference_guides/model_libraries/models_extra/phe.rst:81]: I am not certain myself, but I believe it is a technical term - there is an image in the docs which illustrates it.

Comment (eslickj) [reply docs/reference_guides/model_libraries/models_extra/phe.rst:81]: Ah ok, I confess I didn't look at the images.

**Review by eslickj — APPROVED** [review]

**Pull request merged**